modules/apps/27-interchain-accounts/controller/keeper/account.go

```diff
@@ -9,45 +11,56 @@ import (
 	host "github.com/cosmos/ibc-go/v5/modules/core/24-host"
 )
 
-// RegisterInterchainAccount is the entry point to registering an interchain account:
+// RegisterInterchainAccount registers a new interchain account. The MsgChannelOpenInitResponse is discarded.
+func (k Keeper) RegisterInterchainAccount(ctx sdk.Context, connectionID, owner, version string) error {
```

Comment:
It probably makes sense to leave the previous godoc since devs still might read the godoc for this function